src/django_mysql/models/fields/fixedchar.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any, cast
+
+from django.db.backends.base.base import BaseDatabaseWrapper
+from django.db.models import CharField
+
+from django_mysql.typing import DeconstructResult
+
+
+class FixedCharField(CharField):
+    def __init__(self, length: int, *args: Any, **kwargs: Any) -> None:
```

Comment:
Make the arg kwarg only
```suggestion
    def __init__(self, *args: Any, length: int, **kwargs: Any) -> None:
```